selfdrive/car/gm/values.py

```diff
@@ -113,6 +114,7 @@ def init_make(self, CP: car.CarParams):
     GMCarInfo("GMC Sierra 1500 2020-21", "Driver Alert Package II", video_link="https://youtu.be/5HbNoBLzRwE"),
   ],
   CAR.EQUINOX: GMCarInfo("Chevrolet Equinox 2019-22"),
+  CAR.TRAILBLAZER: GMCarInfo("Chevrolet Trailblazer 2020-22", "Adaptive Cruise Control & LKAS"),
```

Comment:
seems like LKAS is standard, so you only need to list ACC here

Reply:
Thanks, 
I removed "LKAS"